### ISSUE TITLE
`from oml.lightning.entrypoints.validate import pl_val` Error

### ISSUE THREAD

**Issue by PapaMadeleine2022** [OML-Team/open-metric-learning#339]
When I run `from oml.lightning.entrypoints.validate import pl_val`, it shows error:
```
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/conda/envs/oml-py3.7/lib/python3.7/site-packages/oml/lightning/entrypoints/validate.py", line 18, in <module>
    from oml.registry.postprocessors import get_postprocessor_by_cfg
  File "/conda/envs/oml-py3.7/lib/python3.7/site-packages/oml/registry/postprocessors.py", line 7, in <module>
    from oml.retrieval.postprocessors.pairwise import (
  File "/conda/envs/oml-py3.7/lib/python3.7/site-packages/oml/retrieval/postprocessors/pairwise.py", line 11, in <module>
    from oml.inference.pairs import (
  File "/conda/envs/oml-py3.7/lib/python3.7/site-packages/oml/inference/pairs.py", line 6, in <module>
    from oml.datasets.pairs import EmbeddingPairsDataset, ImagePairsDataset
  File "/conda/envs/oml-py3.7/lib/python3.7/site-packages/oml/datasets/pairs.py", line 7, in <module>
    from oml.datasets.list_dataset import ListDataset, TBBoxes
  File "<fstring>", line 1
    (len(bboxes)=)
                ^
SyntaxError: invalid syntax

```

How to fix it ?

**Comment by AlekseySh** [comment]
@PapaMadeleine2022 
Hey, this kind of syntax is only working for python 3.8 and later. I will refactor this part of the code so it does not need 3.8.
You can wait for it or update your python.

**Comment by PapaMadeleine2022** [comment]
@AlekseySh  Get it! thanks

**Comment by AlekseySh** [comment]
@PapaMadeleine2022 I've released a new version of the repo where I fixed the problem above: 0.3.14. It will be available from PIP soon (usually in ~30 minutes)
